### DISCUSSION TITLE
Introdução sem Numeração

### DISCUSSION THREAD

**Discussion by leoneldca** [joaomlourenco/novathesis#197] [Q&A] [answered]
Olá, Boa tarde!
Seria possível que a Introdução não tratada como um capítulo?
Tipo uma página posta logo anteriormente ao primeiro capítulo e que fosse numerada ( digo numeração de página e não de capítulo ) normalmente?

Obrigado

**Comment by b-pereira** [comment]
Na página do Facebook do Novathesis existe a resposta a essa pergunta. Fazendo a pesquisa por "capítulo não numerado" encontras lá o que fazer.

**Comment by b-pereira** [comment, marked as answer]
Assim de cabeça, no ficheiro que tens como introdução colocas \chapter*{Introdução} e logo a seguir \addcontentsline{toc}{chapter}{Introdução} para aparecer no Índice

**Reply by leoneldca** [reply]
Obrigado @b-pereira
Olhei lá na página.
Ficou assim:

\chapter*{State of the Art, Motivation and Objectives}
\phantomsection
\markboth{State of the Art, Motivation and Objectives}{State of the Art, Motivation and Objectives}
\addcontentsline{toc}{chapter}{State of the Art, Motivation and Objectives}

Abraços e obrigado

**Reply by b-pereira** [reply]
Não precisas do \phantomsection
